src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,18 +1,34 @@
 package com.google.sps.meltingpot.data;
 
 import java.util.Date;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.sps.meltingpot.data.DBUtils;
 
 public class Comment {
   public static final String CONTENT_KEY = "content";
   public static final String DATE_KEY = "date";
+  public static final String CREATOR_ID_KEY = "creatorId";
 
   public String content;
   public final Date date;
+  public String creatorId;
 
-  public Comment(String content) {
+  public Comment(String content, String creatorId) {
     this.content = content;
     this.date = new Date();
+    this.creatorId = creatorId;
   }
 
   // TODO: get user display name (or username) from db collection.
+
+  public static boolean createdbyUser(String recipeId, String commentId, String userId) {
+    DocumentReference commentRef = DBUtils.comment(recipeId, commentId);
+    ApiFuture<DocumentSnapshot> commentFuture = commentRef.get();
+    DocumentSnapshot comment = DBUtils.blockOnFuture(commentFuture);
```

Comment:
just a "nit" but I'd roll this into a single line (inlining the vars commentRef and commentFuture), once it's "debugged"

Reply:
Yup, inlined in 13fde54